Description: Add drag and drop support when using alt + tab
 Added to 1:2.30.1-2ubuntu2
Author: Matthias Clasen
Upstream bug report: https://bugzilla.gnome.org/show_bug.cgi?id=135056
Ubuntu bug report: https://bugs.launchpad.net/bugs/111939

Index: mutter-2.91.4/src/core/display.c
===================================================================
--- mutter-2.91.4.orig/src/core/display.c	2010-12-23 10:09:52.846944000 +1100
+++ mutter-2.91.4/src/core/display.c	2010-12-23 10:09:54.436944000 +1100
@@ -3568,7 +3568,7 @@
   meta_display_set_grab_op_cursor (display, screen, op, FALSE, grab_xwindow,
                                    timestamp);
 
-  if (!display->grab_have_pointer)
+  if (!display->grab_have_pointer && !grab_op_is_keyboard (op))
     {
       meta_topic (META_DEBUG_WINDOW_OPS,
                   "XGrabPointer() failed\n");
